script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -781,123 +843,136 @@ It's quite ironic.
 }
 [sha:c6056f56d9d2c246c5fda3a338718449b5405b92]{
 -- Page 698, Offset 16550.
 -- 『かたや田舎豪族の息子、かたや<聖痕|スティグマ>があるだけの女。ともに司教までは上り詰められない同士、慰めあっていたのもおしまいだ。まあいいけどね。
-"On one side, the son of some provincial magnate, on the other, a woman who has naught but the Stigma she bears.
- Yet neither were ultimately able to rise to the position of bishop, instead finding solace in one another. The End. Well, whatever.
+// What's the relationship here? How much should people be encouraged to read into 慰め合う?
+'On one side, the son of some provincial magnate.
+ On the other, a woman who has naught but the Stigma she bears.
+ Two comrades who failed to make it to the rank of bishop together,
+ and who will no longer count on each other for comfort.
+ The End. Well, whatever.
 }
 [sha:28e0daaf4967b2e941fb9d2fc1d7c0d55ff1c462]{
 -- Page 698, Offset 16551.
 -- それで、君は自分の魔術理論を完成させるわけか。あたしはここから離れる気はないから、君に合わせて生きていく事はできんな。ま、幸いあたしは女だ。さっさと子供を産んで君の事を伝えるとしよう』
-So, you'll complete your Magecraft Theory, I take it?
- I don't have any intentions of leaving the Church, so I can't go down the same path as you.
- Though, fortunately I'm a woman.
- Guess I'll get some kids real quick so I can tell them all about you."
+So then, you're still planning on finishing your Magecraft Theory, I take it?
+ I have no intention of leaving the Church, so I can't go down the same path as you.
+ Well, luckily for me, I'm a woman.
+ Guess I'll pop some kids out so I can tell them all about you.'
 }
 [sha:c6076236a3ef0af6d0a0e3e297ba4121d8b89f8f]{
 -- Page 698, Offset 16552. Mods: @n.
 -- 『ほう。なんと伝えますか、
-"Oh? What will you tell them,
+'Oh? And what will you tell them,
 }
 [sha:6df8946baa3d3e9d594cb68f719d1bbff84c0456]{
 -- Page 698, Offset 16553. Glued.
 -- ナルバレック』
-Narbareck?"
+ Narbareck?'
 }
 [sha:90c4997357bfcb09947ac0e63b56369ef9242c57]{
 -- Page 699, Offset 16554.
 -- 『そうだな。あと百年もたてば新参の死徒が台頭してくる。そいつの相手をしても無駄骨だから無視してしまえ、と口伝にしよう』
 // 新参 not referring to the Ancestor sense of the word, so probably best not to use "young blood", but hey, it might fit anyway, I'll think about it.
-"Hmm, I guess something like: 'in a century or so a new Dead Apostle's gonna enter the scene.
- Trying to fight him would be pointless, so just leave him alone.'"
+'Hmm, I guess something about how a new Dead Apostle's gonna make his debut in a century or so?
+ Make sure they understand that trying to fight him would be pointless, and they should just leave him alone.'
 }
 [sha:580e3bea37dd37797328dd236054aee99e21d8b9]{
 -- Page 699, Offset 16555.
 -- 『いえ、百年も待つ必要はありません。貴女が生きているうちには異端審問の対象になるでしょう。この身は最も優れた吸血種に成るのですから、10年もあれば十分です』
-"Not quite――you won't have to wait a century.
+'Oh, you won't have to wait a whole century.
  I suspect I'll become a target for inquisition within your lifetime.
- My body will attain the highest grade for a bloodsucker――ten years will prove plenty."
+ This body of mine will make for a fine bloodsucker――ten years should prove plenty.'
```

Comment:
Maybe 'of mine' is unnecessary. This guy is planning of getting a lot of bodies, so he's more talking about his current one.

Reply:
I just want to note that "This body of mine" never ceases to make me laugh seeing it after Apocrypha.

Good work